### DISCUSSION TITLE
长评论和视频不匹配

### DISCUSSION THREAD

**Discussion by guamian1337** [Richasy/Bili.Uwp#793] [常规]
![Snipaste_2022-03-05_21-21-56](https://user-images.githubusercontent.com/92618050/156885871-a0533981-20fa-4e15-95f8-25a273c31e24.png)
![Snipaste_2022-03-05_21-24-40](https://user-images.githubusercontent.com/92618050/156885874-0e67cbd2-0261-4864-8a8a-c18981f9b432.png)

**Comment by Richasy** [comment]
新版本修复了这个问题
